test/unit-test-helpers/mock_amc.js

```diff
@@ -42,7 +42,8 @@ fake_amc = function() {
     playerSkinPluginsElement : $("<div>", {}),
     adWrapper : $("<div>", {}),
     transitionToAd: function(){},
-    transitionToMainContent: function(){}
+    transitionToMainContent: function(){},
+    createAdVideoElement: function() {}
```

Comment:
There was a recent change that removed the calling of createAdVideoElement by the vast ad manager. Angie and I had a discussion and agreed this should not be an API for the ad plugins to use. Can you try removing this? If this breaks some unit tests, we can merge this branch with master prior to pushing.